manim/__init__.py

```diff
@@ -65,6 +65,7 @@
 
 from .utils.bezier import *
 from .utils.color import *
+from .utils import color as color
```

Comment:
I think this line does not have an effect right now.
`import manim.color as C` is not working, only
import manim.utils.color as C

Reply:
Can you please try `from manim import color as C` instead?

Reply:
Oh, nice, this one is working 
```python
from manim import Scene,Dot
from manim import color as C
class Foo(Scene):
    def construct(self):
        dot = Dot().set_color(C.GREEN)
        self.add(dot)
        self.wait(1)
```